pkg/render/monitor/monitor.go

```diff
@@ -1199,3 +1232,181 @@ func (mc *monitorComponent) serviceMonitorCalicoKubeControllers() *monitoringv1.
 		},
 	}
 }
+
+// externalPrometheusRole creates the permissions for the external prometheus server to scrape ours.
+func (mc *monitorComponent) externalPrometheusRole() client.Object {
+	return &rbacv1.ClusterRole{
+		TypeMeta: metav1.TypeMeta{Kind: "ClusterRole", APIVersion: "rbac.authorization.k8s.io/v1"},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      TigeraExternalPrometheus,
+			Namespace: mc.cfg.Monitor.ExternalPrometheus.Namespace,
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				// When prometheus was first introduced it was accessed through k8s services/proxy and so to this day,
```

Comment:
"When prometheus was first introduced" implies to me that it is no longer accessed through services/proxy?

Reply:
That's right. But the RBAC is still enforced in our backend servers.

Reply:
Which code enforces this? I'm still not quite following - if it's no longer accessed through services/proxy, then why do we need RBAC for that?

Reply:
What would have been best is if we create our own tigera.io rbac resource for authz. However, when we first moved away from using services/proxy, we kept the same RBAC attributes. Today both the tigera/prometheus and tigera/queryserver are still checking that the request is authorized based on these attributes.  I can make a ticket for implementing this.